### PR TITLE
Updating api calls in AutoGPT builder

### PR DESCRIPTION
**Pull request by Bentlybro**
### Background
Updating the api calls in the AutoGPT builder

### PR Quality Scorecard ✨

- [x] Have you used the PR description template? &ensp; `+2 pts`
- [x] Is your pull request atomic, focusing on a single change? &ensp; `+5 pts`
- [ ] Have you linked the GitHub issue(s) that this PR addresses? &ensp; `+5 pts`
- [ ] Have you documented your changes clearly and comprehensively? &ensp; `+5 pts`
- [ ] Have you changed or added a feature? &ensp; `-4 pts`
  - [ ] Have you added/updated corresponding documentation? &ensp; `+4 pts`
  - [ ] Have you added/updated corresponding integration tests? &ensp; `+5 pts`
- [ ] Have you changed the behavior of AutoGPT? &ensp; `-5 pts`
  - [ ] Have you also run `agbenchmark` to verify that these changes do not regress performance? &ensp; `+10 pts`

### CODE REVIEW

File: rnd/autogpt_builder/src/components/CustomNode.tsx [hunk 1]
```diff
@@ -166,6 +166,17 @@ const CustomNode: FC<NodeProps> = ({ data, id }) => {
     }
   };
 
+  const getStatusValue = (statusData: any) => {
```
Comment (aarushik93) [inline rnd/autogpt_builder/src/components/CustomNode.tsx]: you don't need this anymore? 

I think using this still works if you wanted to

**Review by aarushik93 — DISMISSED** [review]
Seems fine, but I left one comment just in case you do want to use that function as it was

File: rnd/autogpt_builder/src/components/CustomNode.tsx (hunk 1 above)
Comment (Bentlybro) [reply rnd/autogpt_builder/src/components/CustomNode.tsx]: yeah this is not needed any more it seems so removed it

**Comment by Bentlybro** [comment]
removed getStatusValue as is not needed any more

**Review by aarushik93 — APPROVED** [review]

**Pull request merged**